src/components/Home/WorksList/Project/Project.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import data from "../../../../data.json";
+import "./Project.css";
+
+export default function Project(props) {
+    let proj;
+    data.forEach((project) => {
+        if (project.title === props.location.pathname.slice(9)) {
+            return (proj = project);
+        }
+    });
+    let { year, title, img, description } = proj;
+
+    return (
+        <div className="project-card-info-container">
```

Comment:
this is not a project card .. the project card is what you have in the home ... this is more of a `project-page-container`